librz/util/table.c

```diff
@@ -247,6 +247,39 @@ RZ_API void rz_table_add_rowf(RzTable *t, const char *fmt, ...) {
 	rz_table_add_row_list(t, list);
 }
 
+RZ_API void rz_table_set_rowf(RzTable *t, const char *fmt, ...) {
```

Comment:
Please add Doxygen docs for this and next function. Also please add the PRINTF_CHECK macro how it's done for other printf-like functions in Rizin - it will allow compiler to check the arguments